Firebase/Messaging/FIRMessagingRemoteNotificationsProxy.m

```diff
@@ -466,39 +392,38 @@ id getNamedPropertyFromObject(id object, NSString *propertyName, Class klass) {
   return property;
 }
 
-#pragma mark - Swizzled Methods
+#pragma mark - UIApplicationDelegate
 
-void FCM_swizzle_appDidReceiveRemoteNotification(id self,
-                                                 SEL _cmd,
-                                                 UIApplication *app,
-                                                 NSDictionary *userInfo) {
+#if TARGET_OS_IOS
```

Comment:
this is not available in tvOS?

Reply:
Good point. It is deprecated on tvOS 10.0 - the lowest version we support. Now I think, maybe we should still support this method? 
@tejasd @ryanwilson @chliangGoogle what do you think?

Reply:
Since the minimum support is tvOS 10.0, and this method is also not recommended in iOS anymore, it's okay not to support it. 

I'm asking because I wasn't sure the swizzler pod is tvOS supported.